src/util.js

```diff
@@ -373,6 +373,31 @@ define([
         return div.html();
     };
 
+    that.dropdownAutocomplete = function (input, sources) {
```

Comment:
Might be nice to add docstrings to `dropdownAutocomplete` and `questionAutocomplete` to make it easier to tell the purpose of each since their names are so similar.